src/store/CurrentAccount/actions.ts

```diff
@@ -0,0 +1,41 @@
+import {CurrentAccountInterface} from "@/interfaces";
+import {SessionStorageKeys} from "sbc-common-components/src/util/constants";
+import {sleep} from "@/utils";
+import currentAccount from "@/store/CurrentAccount/index";
```

Comment:
It feels weird to import from an index file in the current folder.

Reply:
Good call, that line isn't needed.  I'll remove it.